HLTrigger/Configuration/test/cmsDriver.csh

```diff
@@ -49,6 +50,7 @@ set InputLHCRawPIon2 = $InputLHCRawGRun2
 set InputLHCRawPRef2 = $InputLHCRawGRun2
 set InputLHCRawPIon3 = $InputLHCRawGRun3
 set InputLHCRawPRef3 = $InputLHCRawGRun3
+set InputLHCRawSpec3 = /store/data/Run2023D/Commissioning/RAW/v1/000/370/772/00000/f4fb5fb9-c8b8-48cd-9281-c23902260c92.root
```

Comment:
Why not $InputLHCRawGRun3 ?

Reply:
This was one of the files suggested by FOG as used in their integration tests. No other particular reason.

Reply:
I think the GRun3 file would be better as it contains more physics objects firing more of the triggers.